android/src/main/java/io/mosip/tuvali/verifier/GattService.kt

```diff
@@ -7,14 +7,15 @@ import java.util.*
 class GattService {
   //TODO: Update UUIDs as per specification
   companion object {
-    val IDENTITY_CHARACTERISTIC_UUID: UUID = UUID.fromString("00002030-0000-1000-8000-00805f9b34fb")
+    val IDENTIFY_REQ_CHAR_UUID: UUID = UUID.fromString("00002030-0000-1000-8000-00805f9b34fb")
```

Comment:
Inconsistency in Request word. Some names have Request in full form and some have REQ. Please maintain consistency